plasmapy/particles/parsing.py

```diff
@@ -42,7 +45,7 @@ def _create_alias_dicts(Particles: dict) -> (Dict[str, str], Dict[str, str]):
     case_sensitive_aliases = {}
     case_insensitive_aliases = {}
 
-    for symbol in Particles.keys():
+    for symbol in Particles:
```

Comment:
```suggestion
    for symbol in particles:
```

This should be refactored to `particles` to meet PEP8 standards, and to avoid easy confusion with `Particle`.